crate-private type `Outlines<'_>` in public interface

I think there is a bug in 0.1.10:

```
error[E0446]: crate-private type `Outlines<'_>` in public interface
  --> ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/swash-0.1.10/src/scale/cff/mod.rs:24:5
   |
24 | /     pub fn scale(
25 | |         &mut self,
26 | |         outlines: &outlines::Outlines,
27 | |         id: u64,
...  |
32 | |         outline: &mut Outline,
33 | |     ) -> Option<()> {
   | |___________________^ can't leak crate-private type
   |
  ::: ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/swash-0.1.10/src/scale/cff/outlines.rs:39:1
   |
39 |   pub(crate) struct Outlines<'a> {
   |   ------------------------------ `Outlines<'_>` declared as crate-private
```

`Outlines` should be `pub` I guess instead of `pub(crate)`.

--- a/src/scale/cff/outlines.rs
+++ b/src/scale/cff/outlines.rs
@@ -36,7 +36,7 @@ use super::hint::{HintParams, HintState, HintingSink};
 /// [`subfont`](Self::subfont) method to create an appropriately configured
 /// subfont for that glyph.
 #[derive(Clone)]
-pub(crate) struct Outlines<'a> {
+pub struct Outlines<'a> {
     version: Version<'a>,
     top_dict: TopDict<'a>,
     units_per_em: u16,

